### PR TITLE
[RM Ch05] Corrections from GSMA review

### PR DESCRIPTION
**Pull request by walterkozlowski**

### CODE REVIEW

File: doc/ref_model/chapters/chapter05.rst [hunk 1]
```diff
@@ -630,10 +630,9 @@ Compute Resources
 | infra.hw.cpu.cfg.003 | NUMA alignment                  | NUMA alignment enabled and BIOS | N           | Y           |
 |                      |                                 | configured to enable NUMA       |             |             |
 +----------------------+---------------------------------+---------------------------------+-------------+-------------+
-| infra.hw.cpu.cfg.004 | Simultaneous Multithreading     | Y                               | Optional    |             |
-|                      | (SMT) SMT enabled that allows   |                                 |             |             |
-|                      | each core to work multiple      |                                 |             |             |
-|                      | streams of data simultaneously  |                                 |             |             |
+| infra.hw.cpu.cfg.004 | Simultaneous Multithreading     | SMT enabled that allows each    | Y           |Optional     |
```
Comment (pgoyal01) [inline doc/ref_model/chapters/chapter05.rst:633]: SMT, even for High Performance (HP), is usually "Y" -- there are very few HP exceptions where you have the second thread disabled (SMT disabled).

High Performance does require that both threads are allocated to the same workload compute resource, not different workloads.

**Review by pgoyal01 — APPROVED** [review]

**Review by collivier — CHANGES_REQUESTED** [review]
lgtm but they lots of RST issues to fix
https://build.opnfv.org/ci/job/cntt-tox-rm/1749/console

```
./chapters/chapter06.rst:62: D000 Unexpected indentation.
./chapters/chapter06.rst:187: D000 Malformed table.
./chapters/chapter04.rst:267: D000 Malformed table.
./chapters/chapter04.rst:614: D000 Malformed table.
./chapters/chapter04.rst:662: D000 Malformed table.
./chapters/chapter08.rst:386: D000 Inline interpreted text or phrase reference start-string without end-string.
./chapters/chapter08.rst:396: D000 Malformed table.
./chapters/chapter09.rst:386: D000 Malformed table.
./chapters/chapter05.rst:263: D000 Malformed table.
./chapters/chapter03.rst:1243: D000 Malformed table.
./chapters/chapter03.rst:1282: D000 Malformed table.
```

**Comment by collivier** [comment]
@walterkozlowski do u need help to fix this issues?

**Comment by walterkozlowski** [comment]
@collivier, yes please.  Thank you for your offer!

File: doc/ref_model/chapters/chapter05.rst [hunk 2]
```diff
@@ -158,7 +158,7 @@ Virtual Networking Profiles
 | Reference         | Feature              | Type            | Description                              | Capabilities |
 |                   |                      |                 |                                          | Reference    |
 +===================+======================+=================+==========================================+==============+
-| infra.net.cfg.001 | Connection Point     | e.g. virtio1.1  |                                          |              |
+| infra.net.cfg.001 | Connection Point     |IO virtualisation|  eg. virtio1.1                           |              |
```
Comment (pgoyal01) [inline doc/ref_model/chapters/chapter05.rst:161]:
```suggestion
| infra.net.cfg.001 | Connection Point     |IO virtualisation|  e.g., virtio1.1                         |              |
```

File: doc/ref_model/chapters/chapter05.rst [hunk 3]
```diff
@@ -630,10 +630,9 @@ Compute Resources
 | infra.hw.cpu.cfg.003 | NUMA alignment                  | NUMA alignment enabled and BIOS | N           | Y           |
 |                      |                                 | configured to enable NUMA       |             |             |
 +----------------------+---------------------------------+---------------------------------+-------------+-------------+
-| infra.hw.cpu.cfg.004 | Simultaneous Multithreading     | Y                               | Optional    |             |
-|                      | (SMT) SMT enabled that allows   |                                 |             |             |
-|                      | each core to work multiple      |                                 |             |             |
-|                      | streams of data simultaneously  |                                 |             |             |
+| infra.hw.cpu.cfg.004 | Simultaneous Multithreading     | SMT enabled that allows each    | Y           | Y    |
```
Comment (pgoyal01) [inline doc/ref_model/chapters/chapter05.rst:633]: The right boundary marker needs to be aligned

**Review by collivier — APPROVED** [review]
I approved but I think Pankaj's proposal should be committed.

**Comment by walterkozlowski** [comment]
Changed proposed by @pgoyal01 done.

**Review by pgoyal01 — APPROVED** [review]

**Review by ildikov — APPROVED** [review]

**Review by karinesevilla — APPROVED** [review]

**Pull request merged**